sps-team-38/src/main/webapp/script.js

```diff
@@ -18,4 +18,24 @@ function onSignIn(googleUser){
       userEmail = null;
     });
   }
+
+  function getMessagesJSON(){
+    fetch('/data').then(response => response.json()).then((post)=>{
+      const volunteerListElement = document.getElementById('Name',);
+      console.log(post);
+      volunteerListElement.innerHTML = '';
+      post.forEach((post) =>{
+        volunteerListElement.appendChild(createListElement(post.Names));            
+      });
+    }); 
+}
+
+
+function createListElement(text){
+  const liElement = document.createElement('li');
+  liElement.innerText = text;
+  return liElement;
+}
+
+
```

Comment:
This is going to cause a merge conflict with my current pull request

Reply:
So we have to merge yours first? Or

Reply:
I think so.

Reply:
I'll merge it to see.

Reply:
Resolved the merge conflict